dm/dumpling/dumpling.go

```diff
@@ -214,6 +215,10 @@ func (m *Dumpling) Status(_ *binlog.SourceStatus) interface{} {
 	if m.core == nil {
 		return &pb.DumpStatus{}
 	}
+	return m.printStatus()
+}
+
+func (m *Dumpling) printStatus() *pb.DumpStatus {
```

Comment:
I see printStatus and expect its returning is of type error. maybe no need to split a function for Status, or rename it to status like lightning loader.